src/pages/PrintPage.tsx

```diff
@@ -203,15 +203,21 @@ class SummaryPage extends React.Component<RouteComponentProps, State> {
         <Main>
           <MainChild center>
             <Breadcrumbs step={3} />
-            <Prose textCenter className="no-print">
-              <h1 aria-label="Print your ballot.">
+            <Prose textCenter className="no-print" id="audiofocus">
+              <h1 aria-label="Print your official ballot.">
                 Print your official ballot
               </h1>
               <Text narrow>
                 If you have reviewed your selections and you are done voting,
                 you are ready to print your official ballot.
               </Text>
-              <Button primary big onClick={this.showConfirm}>
+              <span aria-label="First, press the down arrow, then" />
```

Comment:
Hmmm… I really need to test this myself. Wondering… can we use the `autoFocus` like in the modal to autoFocus the "Print Ballot" button?